docs/resources/file.md.erb

```diff
@@ -541,3 +543,14 @@ The `exist` matcher tests if the named file exists:
 The `have_mode` matcher tests if a file has a mode assigned to it:
 
     it { should have_mode }
+
+### `more_permissive_than?(mode)`
```

Comment:
The matcher name in the header should be as seen by the user - `be_more_permissive_than`